charts/lakefs/templates/_env.tpl

```diff
@@ -33,15 +33,15 @@ env:
   - name: LAKEFS_USAGE_REPORT_ENABLED
     value: "true"
   {{- if (.Values.fluffy.sso).enabled }}
+  - name: LAKEFS_AUTH_AUTHENTICATION_API_ENDPOINT
+  value: {{ printf "http://%s/api/v1" (include "fluffy.ssoServiceName" .) | quote }}
```

Comment:
That's not "compiling" wrong indent. 
By compiling I mean running `helm template ...` and making sure it tries to render this.